Initilaising variables...
Length of training data: 740501
Traceback (most recent call last):
  File "./othelloCNN8.py", line 140, in <module>
    predictions = network(boards)
  File "/home2/mxjf75/.local/lib/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "./othelloCNN8.py", line 61, in forward
    t = self.fc1(t)
  File "/home2/mxjf75/.local/lib/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/home2/mxjf75/.local/lib/python3.6/site-packages/torch/nn/modules/linear.py", line 87, in forward
    return F.linear(input, self.weight, self.bias)
  File "/home2/mxjf75/.local/lib/python3.6/site-packages/torch/nn/functional.py", line 1370, in linear
    ret = torch.addmm(bias, input, weight.t())
RuntimeError: size mismatch, m1: [200 x 8192], m2: [16384 x 128] at /pytorch/aten/src/THC/generic/THCTensorMathBlas.cu:290
